transcom/mymove
# B 18318 add seed data for multiple moves INT

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3a873935)

## Summary

This work is to populate the dev db with multiple move seed data. Open to suggestions on what to add/modify to make the scenarios realistic.

### How to test

1. Run `make db_dev_fresh` to populate the dev db with data.

2. Poke around in the database, making sure you can see the moves codes below. You can use this query to see all of the moves: `"locator" LIKE 'MMOVE%'`

3. Make sure that the below move groupings are associated with one service member, each having three moves.

4. Login as a customer and verify you can see the following accounts to log in as:
- multiplemoves@HHG_PPM.com
     - Moves: MMOVEA, MMOVEB, MMOVEC
     - Service member affiliation: Army
     - Should have one service member ID for all three moves
- multiplemoves@HHG_PPM_NTS.com
     - Moves: MMOVED, MMOVEE, MMOVEF
     - Service member affiliation: Air force
     - Should have one service member ID for all three moves
- multiplemoves@NTS_HHG.com
     - Moves: MMOVEG, MMOVEH, MMOVEI
     - Service member affiliation: Navy
     - Should have one service member ID for all three moves
- multiplemoves@PPM.com
     - Moves: MMOVEJ, MMOVEK, MMOVEL
     - Service member affiliation: Coast Guard
     - Should have one service member ID for all three moves
![image](https://github.com/transcom/mymove/assets/147537467/261fe75f-dc2e-4c57-b746-ae43f145d6b7)

5. Login as a TOO and click on any of the moves you see with multiple move codes. Ensure you can see the expected shipments for that move code.

6. Login as a service counselor and click on any of the moves you see with multiple move codes. Ensure you can see the expected information for that move code.

Note: I spent quite a bit of time trying to pass in a custom origin duty location and was unable to get it working. These moves are all in KKFA for now. If we want to expand that to other GBLOCs, I will likely need some assistance on figuring that out.
